Lib/test/test_abc.py

```diff
@@ -149,6 +149,25 @@ def foo(): return 4
             self.assertEqual(D.foo(), 4)
             self.assertEqual(D().foo(), 4)
 
+        def test_object_new_with_one_abstractmethod(self):
+            class C(metaclass=abc_ABCMeta):
+                @abc.abstractmethod
+                def method_one():
+                    pass
+            msg = (r"class C with abstract method method_one")
+            self.assertRaisesRegex(TypeError, msg, C)
+
+        def test_object_new_with_many_abstractmethods(self):
+            class C(metaclass=abc_ABCMeta):
+                @abc.abstractmethod
+                def method_one():
+                    pass
+                @abc.abstractmethod
+                def method_two():
+                    pass
+            msg = (r"class C with abstract methods method_one, method_two")
```

Comment:
Minor style comment: empty line missing after class body; unnecessary parens around the msg string.

Reply:
I removed the parenthesis that were wrapping the string.
However, I did not add an empty line after class body as I don't want to break with the style of the file (other test methods don't have line breaks between class definition and test code. Let me know if this is fine.